site/docs/4.1/components/forms.md

```diff
@@ -1268,6 +1268,11 @@ By default, range inputs "snap" to integer values. To change this, you can speci
 
 ### File browser
 
+{% capture callout %}
+Recommanded plugin to animate custom file input: [bs-custom-file-input](https://www.npmjs.com/package/bs-custom-file-input/)
```

Comment:
Should be recommended

Reply:
It's the typo I told you above. `Recommanded` is wrong.

Reply:
Oh ok it was late yesterday sorry 🤣